wallet/src/events.rs

```diff
@@ -97,3 +201,47 @@ pub fn get_events(from: Option<u32>, to: Option<u32>) -> &'static [Event] {
 
     &buffer.as_slice()[from..to]
 }
+
+pub fn get_children() -> Vec<&'static ChildInfo> {
+    storage::get::<ChildList>()
+        .0
+        .values()
+        .map(|x| &x.info)
+        .collect()
+}
+
+pub fn get_child_events(
+    child: &Principal,
+    from: Option<u32>,
+    to: Option<u32>,
+) -> Option<Vec<ChildEvent>> {
+    let buffer = &storage::get::<ChildList>().0.get(child)?.events;
+    let from = from.unwrap_or_else(|| {
+        if buffer.len() <= 20 {
+            0
+        } else {
+            buffer.len() as u32 - 20
+        }
```

Comment:
We should call this behavior out in the .did file, because another reasonable expectation would be that from=None and to=None means return all events.